src/api/routes/index.ts

```diff
@@ -2,7 +2,7 @@ import { Router, Request, Response } from "express";
 import { PORT } from "../../utils/env";
 import { CHECKAPPTOKEN } from "../middlewares/CheckApp";
 
-import User from "./userRoute";
+import User from "./UserRoute";
 import Book from "./bookRoute";
```

Comment:
If you are going for PascalCase, then let it run across, I went for camelCase, so let's stick to one

Reply:
Ohh okay, that's true, then I will resolve that. Thank you.